src/wp-includes/fonts.php

```diff
@@ -51,3 +51,37 @@ function wp_print_font_faces( $fonts = array() ) {
 	$wp_font_face = new WP_Font_Face();
 	$wp_font_face->generate_and_print( $fonts );
 }
+
+/**
+ * Registers a new Font Collection in the Font Library.
+ *
+ * @since 6.4.0
+ *
+ * @param string[] $config {
+ *     Font collection associative array of configuration options.
+ *
+ *     @type string $id  The font collection's unique ID.
+ *     @type string $src The font collection's data JSON file.
+ * }
+ * @return WP_Font_Collection|WP_Error A font collection is it was registered
+ *                                     successfully, else WP_Error.
+ */
+function wp_register_font_collection( $config ) {
+	return WP_Font_Library::register_font_collection( $config );
+}
+
```

Comment:
how do a unregister a font collection.

Reply:
That's not currently possible but that work is already started: https://github.com/WordPress/gutenberg/pull/54701

Reply:
There is now covered by 

https://github.com/WordPress/gutenberg/blob/066f8777b113ca8a71c79deb052d1deb23fe9860/lib/compat/wordpress-6.5/fonts/fonts.php#L147-L149